src/stories/MoleculeFollowerGoal.tsx

```diff
@@ -1,14 +1,8 @@
 import { ComponentProps } from 'react';
-import './button.css';
-import { TwitchProvider } from '../components/TwitchProvider';
-import { FollowerGoal } from '../components/molecules/FollowerGoal';
+import { FollowerGoal } from '../components/molecules';
 
 export const MoleculeFollowerGoalStory = (
```

Comment:
Can we just do this?
```suggestion
export const MoleculeFollowerGoalStory = FollowerGoal;
```

Reply:
We can probably just avoid this entire file if we're just returning the same thing

Reply:
https://storybook.js.org/docs/react/essentials/actions Seems to be related to making a log for actions like onClick when used in the interface. Looks sensible to leave this alone if we make something reliant on actions.